src/aap_eda/api/views/activation.py

```diff
@@ -125,6 +132,53 @@ def create(self, request):
             status=status.HTTP_201_CREATED,
         )
 
+    @extend_schema(
+        request=serializers.ActivationUpdateSerializer,
+        responses={
+            status.HTTP_200_OK: serializers.ActivationReadSerializer,
+            status.HTTP_400_BAD_REQUEST: OpenApiResponse(
+                description="Invalid data to update activation."
+            ),
+            status.HTTP_409_CONFLICT: OpenApiResponse(
+                description="The activation is not allowed to be updated."
+            ),
+        },
+    )
+    def partial_update(self, request, pk):
+        activation = self.get_object()
+        if activation.is_enabled:
```

Comment:
Disabling is an async operation, do you think we should also check the status of the activation?